### PR TITLE
Intgration fixes

### PR DESCRIPTION
**Pull request by AxelGard**
some cleaning and integration fixes

### CODE REVIEW

File: vgc/pipeline.py [hunk 1]
```diff
@@ -27,6 +29,13 @@ def start(self):
         self.view_controller.start()
         self.input_adapter.start()
 
+
+    def set_cromping_point(self, phi, theta):
```
Comment (DennisBerntsson) [inline vgc/pipeline.py:33]: Should be set_cropping point?

File: vgc/pipeline.py [hunk 2]
```diff
@@ -27,6 +29,13 @@ def start(self):
...
+        """" Sets the point for the filter to crop out """
+        # BUG: THIS NEED TO BE FIXED!  what should zoom be???
+        self.camera_filter.update(jaw_in=theta, pitch_in=phi, zoom_in=2)
```
Comment (DennisBerntsson) [inline vgc/pipeline.py:36]: Is it supposed to be "yaw"? I'd rather use my jaw for chewing food then flying a drone :p

**Review by DennisBerntsson — COMMENTED** [review]
Two things that looked odd in pipeline.py. Please review and possibly fix, otherwise it looked good.

**Comment by AxelGard** [comment]
@DennisBerntsson Okej they should be fixed

**Review by DennisBerntsson — APPROVED** [review]

**Pull request merged**